src/aat/resources/features/F-066/S-181.td.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "must return negative response when request contains an invalid event-trigger",
+	"_guid_": "S-181",
+	"_extends_": "F-066_Test_Data_Base",
+
+	"specs": [
+		"an active profile in CCD",
+		"contains an invalid event-trigger",
+		"has the 404 return code"
+	],
+  "request": {
+    "pathVariables": {
+      "cid": "[[DYNAMIC]]",
```

Comment:
same as above

Reply:
Scenario 181 is removed as it was duplicate.